webpack.config.cjs

```diff
@@ -183,56 +184,65 @@ const sharedConfig = {
   },
 };
 
-const storiesEditor = {
-  ...sharedConfig,
-  entry: {
-    'edit-story': './assets/src/edit-story/index.js',
-  },
-  plugins: [
-    ...sharedConfig.plugins,
-    new WebpackBar({
-      name: 'Stories Editor',
-      color: '#fddb33',
-    }),
-  ],
-  optimization: {
-    ...sharedConfig.optimization,
-    splitChunks: {
-      cacheGroups: {
-        stories: {
-          name: 'edit-story',
-          test: /\.css$/,
-          chunks: 'all',
-          enforce: true,
-        },
-      },
-    },
-  },
+// Template for html-webpack-plugin to generate JS/CSS chunk manifests in PHP.
+const templateContent = ({ htmlWebpackPlugin }) => {
+  // Extract filename without extension from arrays of JS and CSS chunks.
+  // E.g. "../css/some-chunk.css" -> "some-chunk"
+  const filenameOf = (pathname) =>
+    pathname.substr(pathname.lastIndexOf('/') + 1);
+
+  const chunkName = htmlWebpackPlugin.options.chunks[0];
+  const omitPrimaryChunk = (f) => f != chunkName;
+
+  const js = htmlWebpackPlugin.files.js
+    .map((pathname) => {
+      const f = filenameOf(pathname);
+      return f.substring(0, f.length - '.js'.length);
+    })
+    .filter(omitPrimaryChunk);
+
+  const css = htmlWebpackPlugin.files.css
+    .map((pathname) => {
+      const f = filenameOf(pathname);
+      return f.substring(0, f.length - '.css'.length);
+    })
+    .filter(omitPrimaryChunk);
+
+  return `<?php return array(
+    'css' => ${JSON.stringify(css)},
+    'js' => ${JSON.stringify(js)});`;
 };
 
-const dashboard = {
+const editorAndDashboard = {
   ...sharedConfig,
   entry: {
+    'edit-story': './assets/src/edit-story/index.js',
     'stories-dashboard': './assets/src/dashboard/index.js',
   },
   plugins: [
     ...sharedConfig.plugins,
     new WebpackBar({
-      name: 'Dashboard',
-      color: '#ade2cd',
+      name: 'Editor & Dashboard',
+    }),
+    new HtmlWebpackPlugin({
+      filename: 'edit-story.chunks.php',
+      inject: false, // Don't inject default <script> tags, etc.
+      minify: false, // PHP not HTML so don't attempt to minify.
+      templateContent,
+      chunks: ['edit-story'],
+    }),
+    new HtmlWebpackPlugin({
+      filename: 'stories-dashboard.chunks.php',
+      inject: false, // Don't inject default <script> tags, etc.
+      minify: false, // PHP not HTML so don't attempt to minify.
+      templateContent,
+      chunks: ['stories-dashboard'],
     }),
   ],
   optimization: {
     ...sharedConfig.optimization,
     splitChunks: {
-      cacheGroups: {
-        stories: {
-          name: 'stories-dashboard',
-          test: /\.css$/,
-          chunks: 'all',
-          enforce: true,
-        },
-      },
+      chunks: 'all',
```

Comment:
Since this results in file names like `vendors~edit-story-981a678a2f0719a0921b` with hashes in them, we need to update `.github/workflows/continuous-integration-bundle-size.yml` accordingly so that this form of names is recognized.

Otherwise we end up with messages like this for every PR, because the file name changes constantly:

<img width="839" alt="Screenshot 2021-03-01 at 11 29 34" src="https://user-images.githubusercontent.com/841956/109484986-66dc1000-7a81-11eb-96ab-5b4c116b33e4.png">

Alternatively, we can use [`splitChunks.name`](https://webpack.js.org/plugins/split-chunks-plugin/#splitchunksname) here to improve the naming so that it matches the existing `*-chunk-*` pattern

Reply:
This was similar to my comment - https://github.com/google/web-stories-wp/pull/6546#issuecomment-787834581

Reply:
👍